server/routes/private.js

```diff
@@ -110,3 +110,48 @@ router.get('/test', async (req, res) => {
 
 module.exports = router
 
+router.post('/api/add-recurring-schedule', async (req, res) => {
+    // Validate appropriate parameters are passed to view workers at each school 
+    const getWorkersForSchool = Joi.object({
+        schoolId: Joi.number().integer().required(), //The student must specify their school ID in order to view the workers
+    });
+
+    const query = req.query ? req.query : {};
+
+    const schoolId = query.schoolId ? query.schoolId : null;
+
+    const { error } = paramSchema.validate({schoolId});
+
+    if (!_.isNil(error)) res.send(error);
+
+    // Attempts to insert the available workers at each school 
+    const isSuccessfullyInserted = await schoolHandler.getWorkerIdsForSchool(schoolId);
+
+    res.send(isSuccessfullyInserted);
+});
+
+router.post('/worker-availability', async (req, res) => {
```

Comment:
Hi Simran,
To reduce unneeded repetition in the code, I would suggest removing lines 133 to 157 as they already exist above on lines 82 to 109.

Reply:
Hi Amy! Thank you for your comments! I've removed lines 133 to 157!